src/main/java/teammates/common/datatransfer/attributes/FeedbackSessionAttributes.java

```diff
@@ -449,37 +429,19 @@ public int compare(FeedbackSessionAttributes session1, FeedbackSessionAttributes
      * method is called with combined feedback sessions from many courses
      */
     public static void sortFeedbackSessionsByCreationTimeDescending(List<FeedbackSessionAttributes> sessions) {
-        Collections.sort(sessions, new Comparator<FeedbackSessionAttributes>() {
-            @Override
-            public int compare(FeedbackSessionAttributes session1, FeedbackSessionAttributes session2) {
-                int result = session2.createdTime.compareTo(session1.createdTime);
-                if (result == 0) {
-                    if (session1.endTime == null || session2.endTime == null) {
-                        if (session1.endTime == null) {
-                            --result;
-                        }
-                        if (session2.endTime == null) {
-                            ++result;
-                        }
+        sessions.sort(Comparator.comparing((FeedbackSessionAttributes session) -> session.createdTime).reversed()
+                .thenComparing((FeedbackSessionAttributes session1, FeedbackSessionAttributes session2) -> {
+                    if (session1.endTime == null) {
```

Comment:
- One missed case here: what happens if both end times are null?
- This block can be shortened to just two returns with ternary used in each return.